recipe/meta.yaml

```diff
@@ -10,29 +10,41 @@ source:
   sha256: 03ff2e1762af388fbbbed1c1ab7f9f1ec006a91da7db3a68b963da6e7795d2ac
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation  -vv
   entry_points:
     - anyconfig_cli = anyconfig.cli:main
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - setuptools
+    - wheel
   run:
+    - python
     - setuptools
-    - python >=3.6
-
+    - Jinja2
+    - PyYAML
+    - jmespath
+    - jsonschema
+    - ruamel.yaml
+    - simplejson
+    - toml
```

Comment:
These are all optional, see https://github.com/ssato/python-anyconfig/blob/RELEASE_0.13.0/setup.cfg#L71-L81. `ruamel.yaml` and `simplejson` are not actual dependencies.

Reply:
without those the tests are failing